packages/docs/src/03-Usage.svench/CustomAction.svelte

```diff
@@ -0,0 +1,11 @@
+<button on:click={doIt}>trigger custom event</button>
```

Comment:
The whole file should go through Prettier, if possible :sweat_smile: 

I would also add a callback based example... `export let onClick`

Reply:
Døn and Døn